test/unit_test/cats_tree_tests.cc

```diff
@@ -156,6 +157,7 @@ BOOST_AUTO_TEST_CASE(otc_algo_learn_1_action)
     expected_learners.end());
 
   destroy_free<test_learner_t>(&base);
+  ec.l.cb.costs.delete_v();
```

Comment:
could delete_label() be of use or is it wired in a non-reusable way?

Reply:
This seems more transparent to me but changed it.